libsrc/gamate/vga.inc

```diff
@@ -0,0 +1,228 @@
+
+; VGA charset for the PC-Engine conio implementation
```

Comment:
Seems like a C&P leftover.

BTW: Was the term 'VGA' correct for the PC-Engine in the first place?